### DISCUSSION TITLE
Modelling detector tilt (or in-plane rotation) φ, in Astra

### DISCUSSION THREAD

**Discussion by ZacharyEBirch** [astra-toolbox/astra-toolbox#447] [Q&A]
Hello,

For 3D geometries, is there a way to model detector rotation error in Astra? If not, could this by handled by applying a rotational transformation to the vector parameters in the projection geometry? I assume that is how the astra_geom_postalignment function works?

Thanks,
Z

**Comment by askorikov** [comment]
Hi! I would suggest using `astra_geom_2vec` to convert the geometry to a [vector representation](https://astra-toolbox.com/docs/geom3d.html#projection-geometries) and apply an appropriate rotation to the vectors describing detector orientation (columns 7:12 in the geom.Vectors array).

**Reply by ZacharyEBirch** [reply]
This worked perfectly, thanks!
